pkg/batch/executor.go

```diff
@@ -28,14 +29,14 @@ func (b BatchFn) Execute() (interface{}, error) {
 type DelayFn func(dur time.Duration)
 
 type Batcher interface {
-	BatchFor(key string, dur time.Duration, exec Executable) (interface{}, error)
+	BatchFor(key string, dur time.Duration, exec Executer) (interface{}, error)
 }
 
 type nonBatchingExecutor struct {
 }
 
-func (n *nonBatchingExecutor) BatchFor(_ string, _ time.Duration, exec Executable) (interface{}, error) {
-	return exec.Execute()
+func (n *nonBatchingExecutor) BatchFor(_ string, _ time.Duration, ft Executer) (interface{}, error) {
```

Comment:
Not a fan of this renaming: it is not clear what "ft" means.  (I would be OK with a single-letter name because the scope is so short, and the single letter clearly means nothing.  Just "`ft`" makes me try to glean a meaning, which is unimportant here.)

Reply:
nice catch thanks! this are the leftovers from the experimental rename I did to fetcher. I changed it back to exec